closure/goog/net/jsloader_test.js

```diff
@@ -118,6 +118,22 @@ function testLoadMany() {
   var testUrls1 = ['testdata/jsloader_test3.js', 'testdata/jsloader_test1.js'];
   var result = goog.net.jsloader.loadMany(testUrls1);
 
+  // Load another set of URLs independent of the default loadMany queue.
+  window.test5 = null;
+  window.test6 = null;
+  var skipQueueTestUrls = ['testdata/jsloader_test5.js',
+      'testdata/jsloader_test6.js'];
+  var options = {
+    skipQueue: true
+  };
+  var skipQueueResult = goog.net.jsloader.loadMany(skipQueueTestUrls, options);
+
+  // Verify that the skipQueueResult is a separate deferred.
+  assertNotEquals('deferreds are not identical', skipQueueResult, result);
+
+  // Check that the independent test was not loaded yet.
+  assertEquals('verification object', null, window.test5);
+
```

Comment:
Yes that should work. Unless you object, I'd still keep 1 loadMany + 1 loadMany(skipQueue) as the testing scenario though, it seems to make most sense.